app/models/service_provider.rb

```diff
@@ -64,6 +64,7 @@ def self.possible_attributes
       phone
       x509_subject
       x509_presented
+      verified_at
```

Comment:
**question:** Do we have any tests for changing redirect_uris? If we do, can we modify the tests to check this off as well (so we implicitly test the fact that this attribute is available)?